### PR TITLE
Add helper CLIs for ERC20

### PR DESCRIPTION
**Pull request by codchen**
## Describe your changes and provide context
Add CLIs so that node admins can view/send ERC20 tokens without having to using MetaMask

## Testing performed to validate your change
local sei
Screen Shot 2024-01-31 at 12 46 21 PM

### CODE REVIEW

File: x/evm/client/cli/tx.go [hunk 1]
```diff
@@ -514,6 +515,70 @@ func CmdCallContract() *cobra.Command {
...
+			}
+
+			fmt.Println("Transaction hash:", resp.Hex())
+			return nil
+		},
+	}
+
+	cmd.Flags().Uint64(FlagGasFeeCap, 1000000000000, "Gas fee cap for the transaction")
+	cmd.Flags().Uint64(FlagGas, 7000000, "Gas limit for the transaction")
+	cmd.Flags().Uint64(FlagEVMChainID, 713715, "EVM chain ID")
```
Comment (yzang2019) [inline x/evm/client/cli/tx.go]: I wonder if we can just query the chainID from the EVM rpc?

Comment (codchen) [reply x/evm/client/cli/tx.go]: yeah, updated

**Review by yzang2019 — APPROVED** [review]

**Comment by Eko101089** [comment]
> ## Jelaskan perubahan Anda dan berikan konteksnya
> Tambahkan CLI sehingga admin node dapat melihat/mengirim token ERC20 tanpa harus menggunakan MetaMask
> 
> ## Pengujian dilakukan untuk memvalidasi perubahan Anda
> sei lokal Tangkapan Layar 31-01-2024 pukul 12 46 21 siang

Come on

**Review by Eko101089 — COMMENTED** [review]
Come on go

**Pull request merged**